File: lista-2/questao-extra/Ator.py
import random


class Ator():

    def __init__(self, nome=0, idade=0):
        self.nome = nome
        self.idade = idade
        self.identificador = random.randint(1, 10000)

    def setNome(self, nome):
        self.nome = nome

    def getNome(self):
        return self.nome

    def setIdade(self, idade):
        self.idade = idade

    def getNome(self):
        return self.nome

    def getIdentificador(self):
        return self.identificador

    def __str__(self):
        string = "identificador: {} | Nome: {} | Idade: {}".format(
            self.identificador, self.nome, self.idade)
        return string

File: lista-2/trab/Questao-1/Ator.java
package dados;

class Ator {
    private String nome;
    private int idade;
    private int id;

    public Ator() {

    }

    public void setId(int id) {
        this.id = id;
    }

    public int getId() {
        return id;
    }

    public void setIdade(int idade) {
        this.idade = idade;
    }

    public int getIdade() {
        return idade;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getNome() {
        return nome;
    }

    @Override
    public boolean equals(Object obj) {
        Ator u = new Ator();

        return u.equals(obj);
    }

    @Override
    public String toString() {
        StringBuffer string = new StringBuffer();
        string.append("Ator: \n");
        string.append("\tId: " + String.valueOf(idade) + "Nome: " + nome + "Idade: " + String.valueOf(idade));
        return string.toString();
    }
}